experimental/dds/tree2/package.json

```diff
@@ -77,7 +77,7 @@
 		"@fluidframework/runtime-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
 		"@fluidframework/runtime-utils": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
 		"@fluidframework/shared-object-base": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
-		"@sinclair/typebox": "^0.25.21",
+		"@sinclair/typebox": "^0.27.8",
```

Comment:
Motivation for bump is [this change](https://github.com/sinclairzx81/typebox/blob/master/changelog/0.26.0.md#intersect-and-union-now-compose), which makes all of the patterns used in change encoding actually work.